rotkehlchen/globaldb/handler.py

```diff
@@ -373,6 +373,8 @@ def search_assets_levenshtein(
                     continue
 
                 substr_in_name = substr_in_symbol = None
+                # set the default to a number higher than the threshold
+                lev_dist_name = lev_dist_symbol = 6
```

Comment:
what does this do exactly? I do not understand. The comment `"set the default to a number higher than the threshold`" does not really explain it sufficiently.

Does this make it so that if either name or symol is missing there is no match?

Reply:
name can be optional as seen here.
https://github.com/rotki/rotki/blob/894f8c68147289991151b3a6189e05a46dba39bd/rotkehlchen/globaldb/handler.py#L378

same thing for symbol
https://github.com/rotki/rotki/blob/894f8c68147289991151b3a6189e05a46dba39bd/rotkehlchen/globaldb/handler.py#L383

here, the Levenshtein distance is calculated but either of those variables might not be available which is the reason for setting `lev_dist_name = lev_dist_symbol = 6`
https://github.com/rotki/rotki/blob/894f8c68147289991151b3a6189e05a46dba39bd/rotkehlchen/globaldb/handler.py#L388

Although, this is accounted for in tests(where I added an asset with only name and another with only symbol). Maybe Python behaviour idk